Library/Homebrew/cmd/update-report.rb

```diff
@@ -428,11 +428,11 @@ def initialize(tap, api_names_txt: nil, api_names_before_txt: nil, api_dir_prefi
       @api_names_before_txt = api_names_before_txt
       @api_dir_prefix = api_dir_prefix
     else
-      initial_revision_var = "HOMEBREW_UPDATE_BEFORE#{tap.repo_var}"
```

Comment:
This look likely to break things as `update.sh` is not using the new format and won't do until a second `brew update` run.

Reply:
The variable name didn't change. Previously `repo_var` would contain the `_` prefix, which is now in the string, i.e. `"#{tap.repo_var}"` -> `"_#{tap.repo_var}"`.

Reply:
This is confusing given that it's used differently in `update.sh` with the same name.

Reply:
In that case, it may make more sense to leave the functionality as is but rename both to `repo_var_suffix`, given it is never actually used as a standalone variable name.

Reply:
@reitermarkus Fine with me to rename both 👍🏻